Remotion/ObjectBinding/Validation.UnitTests/BusinessObjectValidationResultTest.cs

```diff
@@ -280,7 +280,7 @@ public void GetUnhandledValidationFailures_ForBusinessObject_ReturnsFilteredResu
       Assert.That(sortedUnhandledValidationFailures[0].ErrorMessage, Is.EqualTo("Localized Object Validation Error"));
 
       Assert.That(sortedUnhandledValidationFailures[1].ValidatedProperty, Is.SameAs(_customerNumberBusinessObjectProperty));
-      Assert.That(sortedUnhandledValidationFailures[1].ErrorMessage, Is.EqualTo("NotEmptyValidator: Validation error."));
+      Assert.That(sortedUnhandledValidationFailures[1].ErrorMessage, Is.EqualTo("NotEmptyValueValidator: Validation error."));
```

Comment:
Am not 100% sure, but I think we may need to add a test case for NotEmptyListValidator to show every variant is covered?

Reply:
I don't think so, because this test is about creating a `BusinessObjectValidationResult` from a `ValidationResult`, which is independent of the validators that create the `ValidationFailures` in the `ValidationResult`. The test has always used only a small subset of all possible validators.